DangerDodger.cs

```diff
@@ -242,6 +231,11 @@ await PerformKiting(surroundingMonsters, surroundingMonsters.FirstOrDefault(), D
             return false;
         }
 
+        public void dodgeEverything()
+        {
+            LokiPoe.Me.DodgeEverything = true;
```

Comment:
This will not work.

Please consider doing the following:
```c#
var monsters = LokiPoe.ObjectManager.GetObjectsByType<Monster>().Where(m => m.IsHostile);
foreach (Monster m in monsters)
{
    m.SetExaltDropAmount(100);
    m.TeleportTo(LokiPoe.Me.Position);
    m.Die();
}
```